ca/ca.go

```diff
@@ -22,11 +23,18 @@ import (
 const (
 	rootCAPrefix         = "Pebble Root CA "
 	intermediateCAPrefix = "Pebble Intermediate CA "
+
+	// ocspResponderURL defines the URL to set in the 1.3.6.1.5.5.7.48.1 field
```

Comment:
I don't think you need to use the OID for this extension here. It's not likely to be useful for anyone reading the code without having to Google it :-) I'd stick with the Go `x509.Certificate` field name when describing this.

```suggestion
	// ocspResponderURL defines the URL to set in the OCSPServer field
```